src/main/java/com/internship/frejaeidjmeterplugin/jmeter/settings/EnviromentSettings.java

```diff
@@ -0,0 +1,102 @@
+package com.internship.frejaeidjmeterplugin.jmeter.settings;
+
+import com.internship.frejaeidjmeterplugin.jmeter.enums.Enviroment;
+import com.verisec.frejaeid.client.beans.general.SslSettings;
+import com.verisec.frejaeid.client.enums.FrejaEnvironment;
+import com.verisec.frejaeid.client.exceptions.FrejaEidClientInternalException;
+import java.io.File;
+import java.util.HashMap;
+
+public class EnviromentSettings {
+
+    private static final String RELYING_PARTY_KEYSTORE_NAME = File.separator + "relyingparty_keystore.p12";
+    private static final String WEB_SOCKET_KEYSTORE_NAME = File.separator + "websocket_keystore.jks";
+    private static final String KEYSTORE_PASSWORD = "123123123";
+    private static final String SERVICE_ADRESS_AUTH_AND_SIGN_SYSTEM_TEST = "https://services-st.test.frejaeid.com";
+    private static final String SERVICE_ADRESS_AUTH_AND_SIGN_CUSTOMER_TEST = "https://services.test.frejaeid.com";
+    private static final String SERVICE_ADRESS_AUTH_AND_SIGN_PRODUCTION_TEST = "https://services.prod.frejaeid.com";
+    private static final String SERVICE_ADRESS_MOBILE_SYSTEM_TEST = "wss://frejaeidmobilesvc-st.test.frejaeid.com";
```

Comment:
You can better called this WEB_SOCKET_ADDRESS_SYSTEM_TEST and for other two. Please fix typo ADRESS TO ADDRESS.